inyoka_theme_default/templates/planet/index.html

```diff
@@ -16,15 +16,20 @@
 ] %}
 
 {% block planet_content %}
+  {% set rendered_pagination = macros.render_pagination(pagination) %}
+
+  {{ rendered_pagination }}
   {% if days %}
     {% if page == 1 %}
       <div class="planet-newestarticles">
         <h2>{% trans %}Recent articles{% endtrans %}</h2>
-        <ul class="list-inline">
+        <ol class="planet-newestarticles-list">
```

Comment:
as i merged #117, i rebased and squashed the two column-commits, so it should now work without further work